tests/unit_tests/test_wave_06.py

```diff
@@ -110,7 +114,15 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # I wrote these asserts
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert tai.inventory == [item_a, item_b, item_f]
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
Great work! 

Something to consider: L121 and L122 would only pass if Tai's inventory and Jesse's inventory returns items in this order. Do we want to be this strict in our test? If we don't care about order, how else could we check that the right items are in the right inventories?